utils/commits.js

```diff
@@ -1,22 +1,26 @@
 const fetch = require('node-fetch')
 
-const GH_API_PREFIX = 'https://api.github.com'
-const ORGANIZATION_NAME = 'electron'
-const REPO_NAME = 'electron'
-const GITHUB_TOKEN = process.env.GITHUB_TOKEN
-const NUM_SUPPORTED_VERSIONS = 4
-const SLACK_USER = 'releases-wg'
+const {
```

Comment:
Looks like you need a requires to helpers
```suggestion
const {  linkifyPRs, releaseIsDraft } = require('./helpers')

const {
```